ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/timeline/TimelineComponentState.kt

```diff
@@ -86,7 +86,7 @@ internal class TimelineComponentState(
     }
 
     @get:JvmSynthetic
-    val visible by derivedStateOf { presentedPartial?.partial?.visible ?: true }
+    val visible by derivedStateOf { presentedPartial?.partial?.visible ?: style.visible }
```

Comment:
Same thing here in the timeline about maybe filtering out the items that are not visible... Though not sure how this would draw entirely though...